eng/testing/linker/SupportFiles/Directory.Build.props

```diff
@@ -18,5 +18,6 @@
     <TargetOsName Condition=" '$(TargetOsName)' == '' AND $([MSBuild]::IsOSPlatform('FreeBSD'))">freebsd</TargetOsName>
     <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">x64</TargetArchitecture>
     <TargetRuntimeIdentifier Condition="'$(TargetRuntimeIdentifier)' == ''">$(TargetOsName)-$(TargetArchitecture)</TargetRuntimeIdentifier>
+    <DefaultAppHostRuntimeIdentifier Condition=" '$(DotNetBuild)' == 'true' ">$(TargetRuntimeIdentifier)</DefaultAppHostRuntimeIdentifier>
```

Comment:
are you sure this is needed? I think the DotNetBuild property won't be passed through to the linker test projects

Reply:
Without this line, we won't be able to build these projects inside the VMR. This doesn't have a DotNetBuildFromSourceOnly condition as source-build doesn't build tests. For the VMR we want to build tests though.

Agreed, the `DotNetBuild` property is probably not available in that context.

Reply:
OK this was indeed not needed as the `RuntimeIdentifier` in the trimming project already gets set to `TargetRuntimeIdentifier`.